multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -27,9 +27,11 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
We need to delete stale clusters (memberClusterAnnounces were deleted when controller is down) in the ClusterSet at mc-controller restart too?